easybuild/easyconfigs/a/ANTs/ANTs-2.2.0-foss-2016b.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'CMakeMake'
+
+name = 'ANTs'
+version = '2.2.0'
+
+homepage = 'http://stnava.github.io/ANTs/'
+description = """ANTs extracts information from complex datasets that include imaging. ANTs is useful for managing,
+ interpreting and visualizing multidimensional data."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/stnava/ANTs/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['62f8f9ae141cb45025f4bb59277c053acf658d4a3ba868c9e0f609af72e66b4a']
+
+builddependencies = [('CMake', '3.7.1')]
+
+itkver = '4.12.2'
+itkshortver = '.'.join(itkver.split('.')[:2])
+
+dependencies = [
+    ('ITK', itkver, '-Python-2.7.12'),
+    ('VTK', '6.3.0', '-Python-2.7.12'),
```

Comment:
@vanzod I would promote `-Python-2.7.12` to a `versionsuffix` that also applies to `ANTs` itself?